tests/lib/rules/valid-jsdoc.js

```diff
@@ -287,6 +287,22 @@ eslintTester.addRuleTest("lib/rules/valid-jsdoc", {
                 message: "Unexpected @returns tag; function has no return statement.",
                 type: "Block"
             }]
+        },
+        {
+            code: "/**\n * Some text\n *\n * @param arg [optional] text description\n */\nfunction testFunction(arg) {}",
```

Comment:
running a it otherwise results in the described error from #2270 so it does happen here.